lib/hanami/logger.rb

```diff
@@ -142,7 +146,7 @@ def _message_hash(message)
       # @since x.x.x
       # @api private
       def _format(hash)
-        hash.map { |k, v| "#{k}=#{v}" }.join(SEPARATOR)
+        hash.map { |k, v| "#{k}=#{v}" unless v.nil? }.join(SEPARATOR) + NEW_LINE
```

Comment:
can we have a test to assert the case when value is nil? - I don't think it is a bad idea to show `nil` as string. What's the reason behind adding this guard?